wger/settings_global.py

```diff
@@ -195,6 +209,13 @@
 LOGIN_URL = '/user/login'
 LOGIN_REDIRECT_URL = '/'
 
+#social auth keys
+SOCIAL_AUTH_GOOGLE_OAUTH2_KEY = '249957314196-9qustvr303apilj2r7t7o2gi7527ih98.apps.googleusercontent.com'
```

Comment:
Hey Okiror, Great stuff here but since we're training to work on highly sensitive projects, please store the keys in environment variables.

Reply:
Please also consiser telling testers to pip intstall requirements.txt

Reply:
> Hey Okiror, Great stuff here but since we're training to work on highly sensitive projects, please store the keys in environment variables.

thanks for the review @NicholusMuwonge am going to work on that

Reply:
> Hey Okiror, Great stuff here but since we're training to work on highly sensitive projects, please store the keys in environment variables.

@NicholusMuwonge this has been done